assets/scss/application-ie8.scss

```diff
@@ -1,7 +0,0 @@
-$govuk-global-styles: true;
```

Comment:
Govuk-fronend library no longer supports ie8 see 
https://frontend.design-system.service.gov.uk/changes-to-govuk-frontend-v5/#main-changes-to-v5